README.md

```diff
@@ -73,6 +73,7 @@ If you cannot provide a valid website for your company your PR will not be accep
 - [Rigo Technology](https://www.rigonepal.com/) | Lazimpat, Kathmandu | Rigo provides Information Technology Consultancy, System Design and Payroll Outsourcing Service
 
 ## S - U
+- [Sastodeal | Be Genuine](https://www.sastodeal.com) | Tripureshwor, Kathmandu
```

Comment:
Thank you for the PR. Can you also add a one line description of your company? 

And `Be genuine` should probably go in the description.